turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -268,7 +268,11 @@ impl StandardBroadcastRun {
 
         // 2) Convert entries to shreds and coding shreds
         let is_last_in_slot = last_tick_height == bank.max_tick_height();
-        let reference_tick = bank.tick_height() % bank.ticks_per_slot();
+        let reference_tick = if is_last_in_slot {
+            bank.ticks_per_slot() - 1
```

Comment:
comment here would be good

Reply:
Done with aed6df2c52d019f77e7599a850396edf5033e09b, and a bonus comment reference tick only being 6 bits in shred added in c87c79df77c465014adbee5b34aa1e8eedc49640